back/src/adapters/primary/subscribeToEvents.ts

```diff
@@ -76,6 +76,7 @@ const getUseCasesByTopics = (
   ImmersionApplicationCancelled: [
     useCases.broadcastToPoleEmploiOnConventionUpdates,
   ],
+  ImmersionApplicationDeprecated: [],
```

Comment:
Profitons en pour bien nommer: Application est un terme qu'on utilise plus depuis plus d'un an sur l'appli. Il en reste des traces mais on va pas en remettre de nouveaux 😅 . Je te propose : 
```suggestion
  ConventionDeprecated: [],
```